src/widgets/shared-utils.tsx

```diff
@@ -30,6 +30,11 @@ export function openWorkspaceTab<
   }
 }
 
+export function capitalize(s): string {
```

Comment:
This is missing the input type (we should make sure only `string` enter - already from a typing perspective). Likewise, the function already exists in lodash.

Reply:
This function has been removed in the subsequent PR dealing with translations and replaced with the lodash-es equivalent.